### PR TITLE
Django22 support

### PR DESCRIPTION
**Pull request by Ayub-Khan**
**Description:** 
-Removed Django constraint
-updated tox
-updated travis
-moved linting rules to setup.cfg
-updated requirements
-fixed python3.5 tests
-updated trove classifiers
-version bump
-quality improvements

**JIRA:** Link to JIRA ticket
None

**Dependencies:** 
https://github.com/edx/edx-bulk-grades/pull/39

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,28 @@
 language: python
 python:
 - 2.7
+- 3.5
 - 3.6
+- 3.7
```
Comment (awais786) [inline .travis.yml]: we are not using 3.7

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
 env:
-- TOXENV=django111
-- TOXENV=django20
+- TOXENV=django111-drflatest
```
Comment (awais786) [inline .travis.yml]: You have to test the all drf versions also.

Comment (awais786) [reply .travis.yml]: e.g
```
- TOXENV=django111-drf37
- TOXENV=django111-drf38
- TOXENV=django111-drf39
- TOXENV=django111-drflatest
- TOXENV=django20-drf37
- TOXENV=django20-drf38
- TOXENV=django20-drf39
- TOXENV=django20-drflatest
- TOXENV=django21-drf37
- TOXENV=django21-drf38
- TOXENV=django21-drf39
- TOXENV=django21-drflatest
- TOXENV=django22-drf37
- TOXENV=django22-drf38
- TOXENV=django22-drf39
- TOXENV=django22-drflatest
```

File: tox.ini [hunk 3]
```diff
@@ -1,45 +1,20 @@
...
-; D413 = Missing blank line after last section (numpy style)
-; D414 = Section has no content (numpy style)
-ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
-match-dir = (?!migrations)
-
-[pytest]
-DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov super_csv --cov-report term-missing --cov-report xml
-norecursedirs = .* docs requirements
+envlist = py{27,35}-django111-drf{36,37,38,39,latest},py35-django{20,21,22}-drf{36,37,38,39,latest}
```
Comment (jmbowman) [inline tox.ini]: We don't need the drf36 entries, do we?  Not used in Travis, and we're already on 3.7.7 in edx-platform.

**Pull request merged**